bears/generate_package.py

```diff
@@ -37,19 +41,24 @@ def create_file_from_template(template_file, output_file, substitution_dict):
         output_handle.write(template)
 
 
-def create_file_structure_for_packages(root_folder, file_to_copy, object_name):
+def create_file_structure_for_packages(root_folder, file_to_copy,
+                                       object_name, package_type):
     """
     Creates a file structure for the packages to be uploaded. The structure
-    will be ``root_folder/object_name/object_name/file_to_copy``.
-    The file structure has two object_name folders because ``setup.py`` file
+    will be ``root_folder/object_name/object_name/file_to_copy`` for PyPI
+    packages and ``root_folder/object_name/file_to_copy`` for conda packages.
+    The PyPi structure has two object_name folders because ``setup.py`` file
     needs files to be one level deeper than itself so that they can be uploaded.
 
     :param root_folder:  The folder in which the packages are going to be
                          generated.
     :param file_to_copy: The file that is going to be generated the package for.
     :param object_name:  The name of the object that is inside the file_to_copy.
+    :param package_type: The type of package, conda or PyPi.
```

Comment:
should be ``package:`` instead of a comma.

also its PyPI apparently, not PyPi